index.html

```diff
@@ -3874,7 +3874,21 @@ <h3>Syntactic Sugar</h3>
             </li>
           </ul>
         </section>
-
+        <section>
+          <h3>Lists and Arrays</h3>
+          <p>
+Lists and arrays, even lists of lists, are possible when using [[JSON-LD]] 1.1
+as this specification does. We encourage those who want RDF semantics with use
+cases requiring lists and arrays to follow the guidance in
+<a href="https://www.w3.org/TR/json-ld11/#lists">lists in JSON-LD 1.1</a>.
+          </p>
+          <p class="note">
+While it is possible to use this data model without any required JSON-LD
+processing, those who do so and make use of arrays need to be aware that unless
+the above guidance is followed, the order of items in an array cannot be
+guaranteed in JSON-LD. This may lead to unexpected behavior.
```

Comment:
Can the order be guaranteed in a list? If so, we should say so, drawing the clear comparison between arrays and lists.

Reply:
+1 to clarifying the relationship between JSON arrays and JSON-LD lists.

Reply:
best way to do this would be some JSON-LD snippets showing how to define things when order matters.

Reply:
I have added a line in [0fe1a24](https://github.com/w3c/vc-data-model/pull/1258/commits/0fe1a2460d47664a5d1a45ea92d55ed1016308d3) to try and clarify the difference.